utils/releaser/src/FilesProvider/InstallationGuideFilesProvider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\FilesProvider;
+
+use Symfony\Component\Finder\Finder;
+
+final class InstallationGuideFilesProvider
+{
+    /**
+     * @return \Symfony\Component\Finder\SplFileInfo[]
+     */
+    public function provide(): array
+    {
+        $finder = Finder::create()->files()
+            ->name('installation-using*.md')
```

Comment:
so what about production installation document ???

Reply:
it was not in a scope of this PR :innocent:

Reply:
sorry but scope or not scope......THIS IS CRUCIAL